_docs/ops/maintenance-list.md

```diff
@@ -91,57 +91,19 @@ You can also view this information in each of our four environments, `tooling`,
 
 The platform's CI builds can be automatically triggered by third-party resources when updates are released (ie. source code, stemcell). Monitor the `#cg-platform` and `#cg-platform-news` Slack channels for notifications about any failed builds. For any automated builds that failed, review the logs of the failed build task in Concourse, and escalate the failure to platform ops if a cursory triage and rebuild does not fix this issue.
 
-## Review AWS CloudTrail events
+### Monitor cg-provision
 
-### Easy way
+As part of platform automation, and Concourse and Terraform run a `plan-bootstrap-<environment>` job if parts of our infrastructure have changed due to PR or automation. Alerts for reviewing Terraform plan runs are located in `#cg-platoform`.  Review the plan updates and coordinate a time to run `bootstrap-<environment>` so the changes to the infrastructure can be made. All `bootstrap-<environment>` Concourse jobs are set for manual operator release so make sure the review of `plan-bootstrap-<environment>` and running `bootstrap-<environment>` are done in a timely fashion.  This is needed to keep parts of the platform current like platform certificates as example.
 
-Run [cloud-trail-check.sh](https://github.com/cloud-gov/cg-scripts/blob/master/cloudtrail-check.sh) for each AWS account we own,
-and review the output
+### Monitor production deployment jobs
 
-### Hard way
+The Concourse jobs for deploying `production-cf` and `production-logsearch` are setup for manual operator release.  Keep track of when changes are ready to deploy in production since these are customer facing and make sure deployment of these jobs are done during normal platform support hours and during pre-determined times as described [here](https://cloud.gov/docs/overview/customer-service-objectives/). 
 
-> [Get familiar with the documentation for CloudTrail logs](http://docs.aws.amazon.com/IAM/latest/UserGuide/cloudtrail-integration.html).
+## Monitor AWS alerts
 
-Use the [AWS Console](http://docs.aws.amazon.com/govcloud-us/latest/UserGuide/govcloud-console.html)
-to [review API activity history](http://docs.aws.amazon.com/awscloudtrail/latest/userguide/view-cloudtrail-events-console.html)
-for the _EventNames_ listed below.
-Or, use the AWS CLI with the appropriate `$event_name`, and parse the emitted JSON:
+Keep tabs on the private channel `#cg-aws-security` and review the alerts that are posted by AWS GuardDuty.  Review the linked CloudWatch alert for more details and research what triggered the alert.  
```

Comment:
Alerts from GuardDuty go to the Google Group, not the Slack channel

People should also monitor `#cg-aws-security` for the alerts on failed logins, etc

Reply:
ok - misunderstanding - i'll add an additional sub heading for this.

Reply:
@markdboyd changes made in new commit